pages/settings/teams.tsx

```diff
@@ -153,4 +156,15 @@ export default function Teams(props) {
       </SettingsShell>
     </Shell>
   );
-}
\ No newline at end of file
+}
+
+export const getServerSideProps: GetServerSideProps = async (context) => {
```

Comment:
I don't think this is the way to fix this issue; rather, when a 401 is thrown during `loadData` we should catch that 401 and redirect to Login instead.

Reply:
@emrysal Thanks for that. I've updated the code, should be ok now 👍